fe/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
add intellj specific config folder ignores. `.idea` for intellij idk if vsc has something.

Reply:
Done in [7098c9](https://github.com/TPT-Loane/TPT-Loane/pull/1/commits/7098c9066687581136d521354451d2f361002bd7)